client/src/App.js

```diff
@@ -8,13 +8,17 @@ import Tours from "./pages/Tours";
 import ToursToday from "./pages/ToursToday";
 import AddTour from "./pages/AddTour";
 import TourInfo from "./pages/TourInfo";
+import Rider from "./pages/Rider";
 
 function App() {
   return (
     <>
       <GlobalStyle />
       <Router>
         <Switch>
+          <Route path="/riders">
+            <Rider />
```

Comment:
If the route is `riders`, I expect multiple riders in the page.

Rename the page to `Riders` or the route to `/rider` :)

Reply:
Resolved 😄